lib/utils/parseList.js

```diff
@@ -27,7 +29,14 @@ function getParseDetailList (opts) {
         lang: opts.lang,
         country: opts.country,
         cache: opts.cache
-      });
+      })
+        .then((app) => {
```

Comment:
Yes, I think it should be coupled with `fullDetail` and only available if fullDetail is true, seeing that in this case the user is requesting even more details than what would be provided by fullDetail.

Do you think it should work regardless the `fullDetail` value?